.gitignore

```diff
@@ -5,6 +5,7 @@ composer.phar
 /vendor/
 var/
 composer.lock
+.php-cs-fixer.cache
 .php_cs.cache
```

Comment:
Can we drop this line in that case?

Reply:
Aye, will drop it immediately after regression tests pass.